sys/fuchsia/process.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See: https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/process.fidl
+
+include <zircon/syscalls.h>
+
+resource zx_process[zx_handle]
+
+zx_process_exit(ret_code int64)
+
+zx_process_create(job zx_job, name ptr[in, string], name_len len[name], options const[0], proc_handle ptr[out, zx_process], vmar_handle ptr[out, zx_vmar])
+
```

Comment:
This just reminded me to file a tracking bug in monorail for inclusion of `next` syscalls once we have the right job policy in place. Would you be able to add a TODO here with a reference to https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=118199 to add a definition for `zx_process_create_shared`?

Reply:
Done!